Many fixes for China and Japan

BlueFoxyLeMage opened the pull request:
- Reorganized the democratic tree for China, giving it more bonuses
- Changed the effects of spirits and focuses for China
- Fix for the "Restart the Civil War" focus for the PRC
- "Government Reshuffle" and "Boost the Moral" (PRC) are now mutually exclusive
- Fixed Rhee's portrait for Korea
- Fixed the Imperial Federation focus being unavailable because UK is democratic and South Africa is neutral
- Japan can't attack China if they're in a faction with them
- New focus icons for Japan and China
- Fixed small saving problems on Lemon's Hirohito and James Gavin

Commit:
fsq

Commit:
Merge branch 'Beta-Edits' into fixes

Commit:
Zhang will now always accept if you go with the Kuomintang

Commit:
Many things

- Fixed Manchukuo's Royal Guard focus
- Gave some actual buffes to Manchukuo when taking the "Contact Quig Restorationists" route
- New icons for Ireland
- Fixed a missing icon in Portugal's Tree

Commit:
forgot to save one of the icons

Commit:
Added an event if France has selected "Far Right Leagues" or "Action Française" before the Front Populaire is in power, wich instead elects the right

Commit:
Austria can't invite Otto if he's in Hungary and vice versa

Commit:
fixed somethjing with digits

Commit:
PRC white-peace ing Xibei San Ma is now a 50/50 chance rather than a 75/25

Commit:
Fixed Malaysian icons that looked meh

Commit:
fixed more icons

Commit:
Yezhov's Portrait

Commit:
Changed some focus icons and fixed A LOT OF FLAGS

Commit:
Merge branch 'Beta-Edits' into fixes

Commit:
Many fixes

- Reorganized the democratic tree for China, giving it more bonuses
- Changed the effects of spirits and focuses for China
- Fix for the "Restart the Civil War" focus for the PRC
- "Government Reshuffle" and "Boost the Moral" (PRC) are now mutually exclusive
- Fixed Rhee's portrait for Korea
- Fixed the Imperial Federation focus being unavailable because UK is democratic and South Africa is neutral
- Japan can't attack China if they're in a faction with them
- New focus icons for Japan and China
- Fixed small saving problems on Lemon's Hirohito and James Gavin

Commit:
Merge branch 'Beta-Edits' into fixes

Commit:
removed duplicate

Commit:
Fixed Cuban icons and a siamese one and reworked the time for the economic tree of Siam

Commit:
Merge branch 'Beta-Edits' into fixes

Thinking-waffle reviewed (changes requested):
Why did you remove the Greek decision to rename northern Epirus?

Commit:
Revert "removed duplicate"

This reverts commit cdcd770caa2a1b29f7a7a341f9281b2d79a611ea.

Commit:
Prussia decision + focus icons for Austria

Commit:
Many things

- Added allowed = {} to many decisions
- Added a new subideology for neutrality called absolute_monarchism
- Icons for Yugoslavia
